gtsam/hybrid/MixtureFactor.h

```diff
@@ -149,7 +169,7 @@ class MixtureFactor : public HybridFactor {
 
   /// print to stdout
   void print(
-      const std::string& s = "MixtureFactor",
+      const std::string& s = "",
```

Comment:
Why?

Reply:
It anyway prints `MixtureFactor` below on line 176.